src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -128,7 +133,9 @@ export function getTypes(indices) {
 export function getIndices(includeAliases) {
   const ret = [];
   $.each(perIndexTypes, function (index) {
-    ret.push(index);
+    if (!index.startsWith('.ds')) {
```

Comment:
why do we need to ignore hidden datastreams? 🤔

Reply:
If there are not ignored they will show up in the suggested indices. Also, according to this comment from https://github.com/elastic/kibana/issues/73060

> We will probably need to populate them using `GET _data_stream/*` and we should probably not list `.ds*` indices in the suggested indices.


![2022-03-07_09-22](https://user-images.githubusercontent.com/53621505/156967839-adb9ebed-32cb-4383-915a-a4dea65f8001.png)

Reply:
Makes sense, thanks!

Reply:
Thanks for including this logic and for the explanation @mibragimov! I think you're spot-on. In the future I'd love to see answers to these kinds of questions recorded as code comments. That way, future readers of the code who have the same question will be able to find the answer in the code itself instead of having to check the commit history and work their way back to this PR.